mods/sbz_meteorites/mod.conf

```diff
@@ -1,2 +1,2 @@
 name = sbz_meteorites
-depends = sbz_base, sbz_resources, sbz_power
\ No newline at end of file
+depends = sbz_base, sbz_resources, sbz_power, areas
```

Comment:
i don't think having minetest.is_protected requires requiring areas but sure

Reply:
i mean all of the checks happen after load time but... yea